sklearn/feature_selection/from_model.py

```diff
@@ -17,11 +18,21 @@ def _get_feature_importances(estimator):
     importances = getattr(estimator, "feature_importances_", None)
 
     if importances is None and hasattr(estimator, "coef_"):
-        if estimator.coef_.ndim == 1:
-            importances = np.abs(estimator.coef_)
+        if isinstance(estimator.coef_, np.ndarray):
+            if estimator.coef_.ndim == 1:
+                importances = np.abs(estimator.coef_)
+            else:
+                importances = np.sum(np.abs(estimator.coef_), axis=0)
+
+        elif sps.isspmatrix_csr(estimator.coef_):
+            importances = np.sum(np.abs(estimator.coef_.toarray()), axis=0)
```

Comment:
Yeah you might want to copy the ``coef_``, I think. I'm not sure how we usually do that. ``np.abs`` works on scipy sparse matrices, but I'm not entirely sure what it does. I think copy and using data is save. @jnothman ?